[Teams] Update teams-devices-faq.md (#2350)
cloudflare/cloudflare-docs: products/cloudflare-one/src/content/faq/teams-devices-faq.md

Cloudflare WARP is in part powered by 1.1.1.1. When visiting sites or going to a new location on the Internet, you should see blazing fast DNS lookups. However, WARP is built to trade some throughput for enhanced privacy, because it encryps all traffic both to and from your device. While this isn't noticeable at most mobile speeds, on desktop systems in countries where high speed broadband is available, you may notice a drop. We think the tradeoff is worth it though and continue to work on improving performance all over the system.

## Firefox is showing a network protocol violation when I use the WARP client.
If you see this warning, you may have to disable DNS over HTTPs setting in Firefox. If you need help doing that, see [these instructions](https://support.mozilla.org/en-US/kb/firefox-dns-over-https#w_manually-enabling-and-disabling-dns-over-https).

## Why is my device not connecting to the Internet after I deploy WARP?

Deploying new software can come with unexpected issues. This section covers the most common issues you might encounter as you deploy the WARP client in your organization, or turn on new features that interact with the client. 

> ### Is the Cloudflare root certificate installed in all the places?

Reviewer: This and the following questions feel fundamentally different than the other questions on the page. They're a bit more like a `Troubleshooting checklist` than FAQ, if that makes sense.

If that's not what you wanted, I'd suggest rephrasing the questions to be more focused on the customer-observed outcome than the solution. Or maybe you do want this and the content belongs on another page?

Reviewer: Ugh, ignore ^^^, I totally missed that these are nested questions... still drinking my morning coffee!

Installing and trusting the [Cloudflare root cert](/connections/connect-devices/warp/install-cloudflare-cert) is a necessary step to enable advanced security features, such as Browser Isolation, HTTP filtering, AV scanning, and device posture. It is required once you enable the Proxy in **Settings** > **Network** > **HTTP Filtering**.

> In addition to ensuring the root certificate is trusted at the device level, many applications also rely on their own certificate store. Applications like Firefox, Docker, Python, or NPM all rely on their own certificate store and the Cloudflare root certificate must be trusted in each. 

Reviewer: ```suggestion
In addition to ensuring the root certificate is trusted at the device level, many applications also rely on their own certificate store. Applications like Firefox, Docker, Python, and NPM all rely on their own certificate store and the Cloudflare root certificate must be trusted in each. 
```

If you ever see an error like *Certificate not trusted* or *Not trusted identity* or *SSL Error*, it is likely related to our root certificates.

> As a last resort, please add the application to a Do Not Decrypt policy in Gateway.

Reviewer: ```suggestion
As a last resort, add the application to a **Do Not Decrypt** policy in Gateway.
```

> ### Some applications are not compatible with TLS decryption. Have you set up a Do Not Inspect policy for them?

Reviewer: ```suggestion
### Have you set up a Do Not Inspect policy for applications not compatible with TLS decryption?
```

> You may need to set up a Do Not Inspect policy to exclude some applications that do not support SSL inspection, or are otherwise incompatible with TLS decryption, from Gateway inspection. These applications could show errors once the WARP client is connected. For more information, refer to our documentation on [Do Not Decrypt applications](/policies/filtering/http-policies/application-app-types#do-not-decrypt-applications). 

Reviewer: Should you be bolding or italicizing `Do Not Inspect`? Legitimately might be a preference thing for me, but I think I like to do stuff like that.

> ### Do you, your ISP or your country have policies in place that would block the WARP client?

Reviewer: ```suggestion
### Do you, your ISP, or your country have policies in place that would block the WARP client?
```

> Check that you don't have policies or firewall rules in place that block communication to the transport mechanisms or IP addresses required for WARP. Check the requirements at [this page](https://developers.cloudflare.com/cloudflare-one/connections/connect-devices/warp/deployment/firewall).

Reviewer: ```suggestion
Ensure that you do not have policies or firewall rules in place that block communication to the transport mechanisms or IP addresses required for WARP. Check the requirements at [this page](/connections/connect-devices/warp/deployment/firewall).
```

> In particular, note that Microsoft Intune’s default security policy creates a Firewall rule that will block WARP by default. Please see the page linked above for instructions on which IP addresses you need to add.

Reviewer: ```suggestion
In particular, Microsoft Intune’s default security policy creates a Firewall rule that will block WARP by default. Refer to the page linked above for instructions on which IP addresses you need to add.
```

Reviewer: ```suggestion
In particular, note that Microsoft Intune’s default security policy creates a firewall rule that will block WARP by default. Please see the page linked above for instructions on which IP addresses you need to add.
```

> Note that some countries explicitly block the use of VPN or VPN-like software that intentionally encrypts traffic. If you suspect your country may be blocking this traffic, please work with your ISP to verify.

Reviewer: ```suggestion
Some countries explicitly block the use of VPN or VPN-like software that intentionally encrypts traffic. If you suspect your country may be blocking this traffic, please work with your ISP to verify.
```

> ### Are you running another VPN, Firewall or security product that may be trying to inspect traffic, interfere with routing or enforce DNS policy?

Reviewer: ```suggestion
### Are you running another VPN, firewall, or security product that may be trying to inspect traffic, interfere with routing or enforce DNS policy?
```

Running VPNs or firewalls alongside the WARP client may interfere with some of its functionalities. Please refer to the following documentation on how to use WARP alongside a VPN, or on how to exclude traffic from the WARP client:

>  [❯ Use WARP alongside a VPN](/connections/connect-devices/warp/exclude-traffic#use-warp-alongside-a-vpn)

Reviewer: What's the purpose of the `>`? As long as it renders fine – don't want to build the cf1 docs for 40 min, sorry! — I guess it doesn't matter.

 [❯ Exclude traffic from WARP](/connections/connect-devices/warp/exclude-traffic)

The most common places we see interference with WARP from these products are:

> * **Who has control of the routing table.** In the default configuration of WARP where exclude-split tunnel rules are in place, WARP needs control over the default route. Third party VPNs need to be set to only include just routes to your internal resource.

Reviewer: ```suggestion
* **Control of the routing table:** In the default configuration of WARP where exclude-split tunnel rules are in place, WARP needs control over the default route. Third-party VPNs need to be set to only include routes to your internal resource.
```

> * **Who has control of DNS.** WARP Must be the last client to touch the primary and secondary DNS server on the default interface. Make sure any DNS setting is disabled in third-party VPNs.

Reviewer: ```suggestion
* **Control of DNS:** WARP must be the last client to touch the primary and secondary DNS server on the default interface. Make sure any DNS setting is disabled in third-party VPNs.
```

* If running alongside a third-party VPN, you must create an exclude [Split Tunnel rule](/connections/connect-devices/warp/exclude-traffic/split-tunnels) for the VPN server you are connecting to (for example, `vpnserver.3rdpartyvpn.example.com`).

> ### As a last resort, does our consumer application work or does your configuration work on a clean machine?

Reviewer: ```suggestion
### As a last resort, does WARP or your configuration work on a clean machine?
```

> A good first step is to verify that our consumer application works in your environment. This will rule out any issues with policies created in the Teams configuration that could be impacting your connection. You can download the WARP consumer application at `https://1.1.1.1/`.

Reviewer: `First step` feels weird to say when the question explicitly says `Last resort`

Reviewer 2: This is my bad. I changed the order of this question but never changed the opening sentence. Alice maybe something like:

To eliminate issues related to a particular Cloudflare for Teams configuration, rule or policy you can try running WARP in consumer mode. This will allow us to ensure basic connectivity can be achieved. Once this is verified we can slowly start applying additional complexity and testing along the way.

Author: done!
